docs/website/docs/v1.x/dsl/socket/websocketframe.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "WebSocketFrame"
+sidebar_label: "WebSocketFrame"
+---
+In the [WebSocket](https://datatracker.ietf.org/doc/html/rfc6455) protocol, communication happens using frames. ZIO
+HTTP's [WebSocketFrame](https://github.com/dream11/zio-http/blob/main/zio-http/src/main/scala/zhttp/socket/WebSocketFrame.scala)
+is its representation of those frames. The protocol defines the following types of frames, which are generally divided
+into two categories - **Non-Control** and **Control** Frames:
```

Comment:
I think that will be enough to specify: 

`The protocol defines the following types of frames: ` instead of
`The protocol defines the following types of frames, which are generally divided
into two categories - **Non-Control** and **Control** Frames:`

Reply:
Sure thing. Thanks for the review, @devsprint